### PR TITLE
US19 boucle testimony on carousel

### PR DESCRIPTION
**Pull request by khadimb**

### CODE REVIEW

**Review by Mcbess-san — APPROVED** [review]

File: src/View/Visitor/Home/Component/_carousel.html.twig [hunk 1]
```diff
@@ -2,29 +2,19 @@
...
+            {% for testimony in testimonies %}
+                {% if loop.index == 1 %}
+                <div class="carousel-item active text-left p-2">
+                {% else %}
+                <div class="carousel-item text-left p-2">
+                {% endif %}
+                    <p>{{ testimony.message }}</p>
+                    <h3 class="testifying">{{ testimony.name }}</h3>            
+                </div>
+            {% endfor %}
```
Comment (LucyMerlier) [inline src/View/Visitor/Home/Component/_carousel.html.twig:14]: indentation

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comment

**Review by RaphaelBS-WCS — DISMISSED** [review]
Approved

**Review by LucyMerlier — APPROVED** [review]

**Review by RaphaelBS-WCS — APPROVED** [review]
Approved

**Pull request merged**